openstack/datadog_checks/openstack/data/conf.yaml.example

```diff
@@ -126,25 +126,25 @@ instances:
     #   - <KEY_1>:<VALUE_1>
     #   - <KEY_2>:<VALUE_2>
 
-    ## Log Section
-    ##
-    ## type - required - Type of log input source (tcp / udp / file / windows_event)
-    ## port / path / channel_path - required - Set port if type is tcp or udp.
-    ##                                         Set path if type is file.
-    ##                                         Set channel_path if type is windows_event.
-    ## source  - required - Attribute that defines which Integration sent the logs.
-    ## encoding - optional - For file specifies the file encoding, default is utf-8, other
-    ##                       possible values are utf-16-le and utf-16-be.
-    ## service - optional - The name of the service that generates the log.
-    ##                      Overrides any `service` defined in the `init_config` section.
-    ## tags - optional - Add tags to the collected logs.
-    ##
-    ## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
-    #
-    # logs:
-    #   - type: file
-    #     path: /var/log/apache2/*.log
-    #     source: openstack
-    #   - type: file
-    #     path: /var/log/libvirt/*.log
-    #     source: openstack
+## Log Section
+##
+## type - required - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - required - Set port if type is tcp or udp.
+##                                         Set path if type is file.
+##                                         Set channel_path if type is windows_event.
+## source  - required - Attribute that defines which Integration sent the logs.
```

Comment:
```suggestion
## source  - required - Attribute that defines which integration sent the logs.
```

Reply:
to be addressed in https://github.com/DataDog/integrations-core/pull/9764 instead